locks.go

```diff
@@ -66,7 +66,7 @@ func (l *LockEtcd) Lock(ctx context.Context) error {
 }
 
 // Unlock unlocks the previously locked lock.
-func (l *LockEtcd) Unlock() error {
+func (l *LockEtcd) Unlock(_ context.Context) error {
```

Comment:
It makes sense to use this context in the Unlock() operation below (instead of using the `l.cli.Ctx()`).